mkmapdata/download_grids.sh

```diff
@@ -34,8 +35,7 @@ for file in "${files[@]}"; do
   url="${SVNBASE}/${file}"
   if [ ! -f "$localf" ]; then
     echo "Downloading $file ..."
-    # wget -O "$localf" "$url" || { echo "ERROR downloading $url"; exit 1; }
```

Comment:
@mvhulten Thanks for the polishing in general! Only point: The commented-out `wget` command, I actually left on purpose in order to have a link back to the wget comamnd used with `clm_mappingfiles.txt`, envisioning that in the future, the two ways of downloading could be unified.

Reply:
Readded in 3610e4466e3aa5cdd816d0a914d88f3157cdffdf